src/Homework5/get_ranges.py

```diff
@@ -0,0 +1,28 @@
+"""Реализовать функцию get_ranges которая получает на вход
+непустой список неповторяющихся целых чисел, отсортированных
+по возрастанию, которая этот список “сворачивает”
+ get_ranges([0, 1, 2, 3, 4, 7, 8, 10]) // "0-4,7-8,10"
+ get_ranges([4,7,10]) // "4,7,10"
+ get_ranges([2, 3, 8, 9]) // "2-3,8-9"
+"""
+
+
+def get_ranges(lst):
+    temp_list = []
+    ranges_list = []
+    for i in lst:
+        if i + 1 not in lst or i - 1 not in lst:
```

Comment:
O(n**2) solution it can be solved for O(n) complexity

Reply:
rewrited